config/ftbquests/quests/chapters/mekanism.snbt

```diff
@@ -676,19 +674,22 @@
 			]
 		}
 		{
-			title: "Ore Tripling"
+			title: "3rd Tier Ore Processing"
 			icon: "mekanism:electrolytic_core"
 			x: 0.0d
 			y: 4.0d
 			subtitle: "Not Enough Minerals"
 			description: [
-				"With enough effort, Mekanism allows for the extraction of up to five ingots per ore, but this is a very involved process and will require serious effort to set up. "
+				"With enough effort, Mekanism allows for the extraction of up to 3 (3.33 exactly) ingots per ore, but this is a very involved process and will require serious effort to set up."
```

Comment:
This was talking about the absolute cap on mek processing, if that's still 5x, it should still mention it. Here would be a good place to really spell out that Raw Ore and Ore Blocks have different rates with Raw Ore being less and just leave it at that. Let them discover the exact rates through JEI.